classes/components/listPanels/JatsListPanel.php

```diff
@@ -0,0 +1,98 @@
+ */
+
+namespace PKP\components\listPanels;
+
+use APP\core\Application;
+use APP\facades\Repo;
+use APP\publication\Publication;
+use APP\submission\Submission;
+use PKP\components\forms\publication\ContributorForm;
+use PKP\context\Context;
+use PKP\submissionFile\SubmissionFile;
+
+class JatsListPanel extends ListPanel
+{
+    public Submission $submission;
+    public Publication $publication;
+    public Context $context;
+    public array $locales;
+
+    /** Whether the user can edit the current publication */
+    public bool $canEditPublication;
+
+    public function __construct(
+        string $id,
+        string $title,
+        Submission $submission,
+        Context $context,
+        array $locales,
+        bool $canEditPublication = false,
+        Publication $publication
+    ) {
+        parent::__construct($id, $title);
+        $this->submission = $submission;
+        $this->context = $context;
+        $this->locales = $locales;
+        $this->canEditPublication = $canEditPublication;
+        $this->publication = $publication;
+    }
+
+    /**
+     * @copydoc ListPanel::getConfig()
+     */
+    public function getConfig()
+    {
+        $config = parent::getConfig();
+
+        // Remove some props not used in this list panel
+        unset($config['description']);
+        unset($config['expanded']);
+        unset($config['headingLevel']);
+
+        $config = array_merge(
+            $config,
+            [
+                'canEditPublication' => $this->canEditPublication,
+                'publicationApiUrlFormat' => $this->getPublicationUrlFormat(),
+                'uploadProgressLabel' => __('submission.upload.percentComplete'),
+                'fileStage' => SubmissionFile::SUBMISSION_FILE_JATS,
+                'i18nConfirmDeleteFileTitle' => __('publication.jats.confirmDeleteFileTitle'),
+                'i18nDeleteFileMessage' => __('publication.jats.confirmDeleteFileMessage'),
+                'i18nConfirmDeleteFileButton' => __('publication.jats.confirmDeleteFileButton'),
+                'i18nLastModifiedAt' => __('publication.jats.lastModified'),
+                'downloadDefaultJatsFileName' => Repo::submissionFile()->getDefaultJatsFileName($this->submission->getId(), $this->publication->getId()),
+            ]
+        );
+
+        return $config;
+    }
+
+    /**
+     * Get an example of the url to a publication's API endpoint,
+     * with a placeholder instead of the publication id, eg:
+     *
+     * http://example.org/api/v1/submissions/1/publications/__publicationId__
+     */
+    protected function getPublicationUrlFormat(): string
+    {
+        return Application::get()->getRequest()->getDispatcher()->url(
+            Application::get()->getRequest(),
+            Application::ROUTE_API,
+            $this->context->getPath(),
+            'submissions/' . $this->submission->getId() . '/publications/__publicationId__/jats'
```

Comment:
Just for consistency's sake, use `{$publicationId}` as the replacement slug. (also at https://github.com/pkp/ui-library/pull/300/files#diff-564efc80c3d827c62c2b608403f1b0363b0a56c98aae4053ad745001ff1a7c70R182)